doc/sailfish-webview-webview.qdoc

```diff
@@ -6,12 +6,139 @@
+
+  The \c{WebView} component allows you to add web content to a QML page.
 
   The \c{WebView} QML type is exposed through the \c{Sailfish.WebView} QML
   import namespace.
+
+  Exaple of use:
+
+  \code
+  import QtQuick 2.0
+  import Sailfish.Silica 1.0
+  import Sailfish.WebView 1.0
+
+  ApplicationWindow {
+      id: root
+      initialPage: Component {
+          Page {
+              id: page
+              WebView {
+                  anchors.fill: parent
+                  active: true
+                  url: "http://www.sailfishos.org"
+              }
+          }
+      }
+  }
+  \endcode
+
+  \sa WebVewPage
+*/
+
+/*!
+  \qmlproperty WebViewPage webViewPage
+  \brief The \l{WebViewPage} the \l{WebView} is embedded in.
+
+  This will be populated automatically unless explicitly set.
+*/
+
+/*!
+  \qmlproperty bool WebView::canShowSelectionMarkers
+  \brief Display markers when the user selects text on a page.
+
+  If \c{true} the user will be able to select text and items on a page,
+  indicated with handles. Set to \c{false} to disable the selection markers.
+
+  The default is \c{true}.
+*/
+
+/*!
+  \qmlproperty bool WebView::textSelectionActive
+  \brief Indicates that text is currently being selected by the user.
+
+  This read only property will be set to \c{true} if the user is currently
+  selecting text on the page, or \c{false} otherwise.
+*/
+
+/*!
+  \qmlproperty string WebView::httpUserAgent
+  \brief The user agent string to be used for http/https requests.
+
+  Set this value to override the user agent string that will be used for http
+  and https requests. This will be sent in the \c{"User-Agent"} header field.
+
+  Setting this value overrides all other user agent configurations.
+
+  If the string is left empty or cleared, it will get set to the actual user
+  agent string used at the start of each request. If there are domain
+  overrides configured the value may be dependent on the domain being accessed
+  and so may change again on future requests.
+
+  Changing the value mid-request will not alter the behaviour for the onging
+  request, but will affect any future requests.
+
+  Defaults to an empty string.
 */
 
+/*!
+  \qmlsignal WebView::linkClicked(string url)
+  \brief This signal is emitted when the user selects a link on the page.
+
+  When emitted \a url will be set to href selected.
+*/
+
+/*!
+  \qmlmethod WebView::clearSelection()
+  \brief Clears any text currently selected by the user.
+*/
+
+/*!
+  \qmlmethod WebView::runJavaScript(string script, variant callback, variant errorCallback)
+  \brief Runs javascript snippet in the context of the loaded DOM document.
+
+  When the \a script returns, \a callback with a single argument that is the return value is called. The
+  \a callback argument is optional, if set to \c undefined (the default), the return value will be
+  discarded. If the function fails \a errorCallback is called if it is not set to \c undefined
+  (the default).
```

Comment:
The wording, which is a bit tricky, is a little confusing for me concerning the default `errorCallback` value (is it "is not set to undefined (the default)" or "set to undefined (the default)"?). I'd suggest expanding on this in the same way you did for `callback` to make it clearer (and also to explain about the argument to the error function). Something like:
```suggestion
  discarded.
  
  If the function fails \a errorCallback is called with a single argument containing the error 
  string. The \a errorCallback argument is optional, if set to \c undefined (the default), any 
  errors are silently ignored.
```